_posts/2022-04-06-T2T-human-genome.md

```diff
@@ -0,0 +1,30 @@
+---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+title: 'T2T-CHM13v2.0 available in Galaxy'
```

Comment:
The title should probably more telling. People usually do not know the acronym. Complete human genome () available.... Or something like that?

Reply:
> I think the core part regarding Galaxy is still lacking. I would add a final paragraph on where/how that genome is now available (for which tools/from which datatables)?

Sure, I'll add it.